pygac/pygac_geotiepoints.py

```diff
@@ -21,21 +21,22 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-"""Interpolation of geographical tiepoints.
-"""
+"""Interpolation of geographical tiepoints."""
 
 
 import geotiepoints as gtp
 
 import numpy as np
 
 
-
 def Gac_Lat_Lon_Interpolator(lons_subset, lats_subset):
-    """Interpolate lat-lon values in the AVHRR GAC data from every eigth pixel to all pixels.
+    """Interpolate lat-lon values in the AVHRR GAC data.
+
+    Each GAC row has total 409 pixels.
+    But lat-lon values are provided for every eigth pixel.
+    They start at from pixel 5 and at pixel 405.
```

Comment:
Typo, what about `..., ranging from pixel 5 to 405`?

Reply:
Thanks!  I updated the text.